### ISSUE TITLE
Adding sparse gradients

### ISSUE THREAD

**Issue by SirRob1997** [jinmel/lookahead-tensorflow#3]
Hi,

this implementation is exactly what I needed for my purpose. It runs on tf `1.15` and extends the `tf.train.Optimizer `. I was looking at other implementations but failed to get how to properly properly adapt their implemented sparse gradient update into this version, any suggestions?

Best,
Robin

**Comment by SirRob1997** [comment]
The implementation by the authors https://github.com/michaelrzhang/lookahead/blob/master/lookahead_tensorflow.py seems to just reroute the sparse gradient method to the fast optimizer sparse gradient method which worked very poorly for me.
